### PR TITLE
Feat/cdn breadcrumb

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/solarsteein-dedicated`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#15495.0 ✘  
* 
  * Build ✘

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/domains/index.js [hunk 1]
```diff
@@ -4,18 +4,39 @@ import 'oclazyload';
...
+        url: '/domain',
+        redirectTo: 'app.networks.cdn.dedicated.manage.domain.index',
+        views: {
+          cdnView: {
+            template: '<div ui-view></div>',
+          },
+        },
+        resolve: {
+          breadcrumb: /* @ngInject */ ($translate) =>
+            $translate('cdn_dedicated_domains'),
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/domains/index.js:21]:
```suggestion
            $translate.instant('cdn_dedicated_domains'),
```

**Comment by ovh-cds** [comment]
CDS Report build#15503.0 ✘  
* 
  * Build ✘

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/domains/index.js [hunk 2]
```diff
@@ -4,18 +4,39 @@ import 'oclazyload';
...
+
+      $stateProvider.state(
+        'app.networks.cdn.dedicated.manage.domain.index.**',
+        {
+          url: '',
+          lazyLoad: ($transition$) => {
+            const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+
+            return import('./domains.module').then((mod) =>
+              $ocLazyLoad.inject(mod.default || mod),
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/domains/index.js:33]: 👍

**Review by radireddy — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/domains/index.js (hunk 2 above)
Comment (radireddy) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/domains/index.js:33]: @marie-j I had done repsac migration for CDA. QA was also doe but its not merged to develop. I will release my branch on this and submit for QA again.

**Comment by radireddy** [comment]
@marie-j I had done repsac migration for CDA. QA was also doe but its not merged to develop. I will release my branch on this and submit for QA again.

**Comment by marie-j** [comment]
> @marie-j I had done repsac migration for CDA. QA was also doe but its not merged to develop. I will release my branch on this and submit for QA again.

@radireddy CDN ≠ CDA so I don't think it is required

**Review by antleblanc — APPROVED** [review]

**Pull request merged**